public/promises.json

```diff
@@ -0,0 +1,78 @@
+{
+  "promises": [
+    {
+      "title": "Health",
+      "description": "Lorem ipsum dolor sit, amet consectetur adipisicing elit. Non maiores provident numquam adipisci illum facere doloribus quasi eos sapiente similique! Enim accusantium, eaque laboriosam et nemo eligendi officiis aliquam totam adipisci reiciendis eos ipsam, voluptatum quo suscipit modi odio beatae exercitationem ut! Velit rem saepe quod modi distinctio incidunt cupiditate?",
+      "status": { "title": "In Progress" },
+      "content": "By 2022 more than 13 million Kenyans will have health insurance through NHIF",
+      "date": "29/07/2021",
+      "image": "https://africacheck.org/sites/default/files/2018/05/Jubilee_party.jpg",
+      "tags": ["war", "health"],
+      "category": "health",
+      "slug": "health",
+      "href": "/promises/1/health",
+      "id": 1,
+      "key_promise": "True",
+      "relatedPromises": [
+        {
+          "id": 2,
+          "tags": ["health", "war"],
+          "title": "Build School",
+          "slug": "Build-School",
+          "href": "/promises/2/Build-School",
+          "date": "29/07/2021",
+          "status": { "title": "Complete" },
+          "image": "https://upload.wikimedia.org/wikipedia/commons/d/dd/Uhuru_Kenyatta.jpg"
+        }
+      ],
```

Comment:
relatedPromises will have to "computed" based on tags/categories/etc.

Reply:
@kilemensi concerning this, 
if 2 promises have the same tags/categories then they should be treated as related? Or are there minimum similarities that should we use?

Reply:
Nah, I think we should keep it as simple as possible for now... but yes, the more tags/categories they share the better/more related @esirK